nabclockd/nabclockd.py

```diff
@@ -52,11 +52,12 @@ def synchronized_since_boot(self):
         synchronized_date = subprocess.run(
             ["stat", "-c", "%y", "/run/systemd/timesync/synchronized"],
             stdout=subprocess.PIPE,
+            stderr=subprocess.DEVNULL,
```

Comment:
This is a separate, albeit welcome, change

Reply:
Add explicit mention of change in PR description.

Reply:
Could you please file a separate PR?

Reply:
I'm not willing to do this for a one-line change, which moreover may be considered part of the nabclockd synchronisation date check/logging fix.